docs/source/developer/contributing.rst

```diff
@@ -76,8 +83,78 @@ bot <https://github.com/apps/lock>`__ will lock the issue. If additional
 discussion is desired, or if the pull request doesn't fully address the
 locked issue, please open a new issue referencing the locked issue.
 
-Tag Issues with Labels
-^^^^^^^^^^^^^^^^^^^^^^
+New issues are subject to triage. A developer will triage your new issue 
+by doing the following:
+
+1. Read the issue
+2. Search the existing issues and mark it as a duplicate if necessary
+3. If additional information is required, add a comment requesting it
+4. If the issue is ready to be worked on, assign it to a milestone
+5. Apply appropriate labels to the issue (see examples below)
+
+You may start to work on your issue as soon as you file it. Please 
+work with a triager if they have any questions about your issue so that 
+your changes can be merged in without delay.
+
+Definition of Ready
+^^^^^^^^^^^^^^^^^^^
+
+**All requested information, where applicable, is provided.** From the 
```

Comment:
This heading needs a bit more context, maybe an intro paragraph of sentence along the lines of.

"One of the main goals of triage is to get issues into a state where they are Ready for someone to work on. This section details what is involved in an issue being Ready for work"